tools/rss/rss.BUILD

```diff
@@ -0,0 +1,69 @@
+cc_library(
+    name = "ad_rss",
+    srcs = glob([
+        "ad_rss/generated/src/**/*.cpp",
+        "ad_rss/impl/src/**/*.cpp",
+        "ad_rss/impl/src/**/*.hpp",
+    ]),
+    hdrs = glob([
+        "ad_rss/generated/include/**/*.hpp",
+        "ad_rss/impl/include/**/*.hpp",
+    ]),
+    includes = [
+        "ad_rss/generated/include",
+        "ad_rss/impl/include",
+    ],
+    linkopts = ["-pthread"],
+    visibility = ["//visibility:public"],
+    deps = ["@map_support//:ad_physics", "@boost//:geometry"],
+)
+
+# cc_test(
+#     name = "ad_rss_test",
+#     srcs = glob([
+#         "ad_rss/impl/test/**/*.cpp",
+#         "ad_rss/impl/test/**/*.hpp",
+#     ]),
+#     includes=["ad_rss/impl/test/test_support"],
+#     deps = [
+#         ":ad_rss",
+#         "@gtest//:gtest_main",
+#     ],
+# )
+
+cc_library(
+    name = "ad_rss_map_integration",
+    srcs = glob([
+        "ad_rss_map_integration/generated/src/**/*.cpp",
+        "ad_rss_map_integration/impl/src/**/*.cpp",
+        "ad_rss_map_integration/impl/src/**/*.hpp",
+    ]),
+    hdrs = glob([
+        "ad_rss_map_integration/generated/include/**/*.hpp",
+        "ad_rss_map_integration/impl/include/**/*.hpp",
+    ]),
+    includes = [
+        "ad_rss_map_integration/generated/include",
+        "ad_rss_map_integration/impl/include",
+    ],
+    linkopts = ["-pthread"],
```

Comment:
What is pthread required for? Rss using some multi threading mechanisms?

Reply:
No, they don't use threading. I will now remove all those pthread flag.